kernel/src/process/process/mod.rs

```diff
@@ -210,6 +213,13 @@ impl Process {
 
         let prof_clock = ProfClock::new();
 
+        let oom_score_adj = AtomicI16::new(
+            parent
+                .upgrade()
+                .map(|parent: Arc<Process>| parent.oom_score_adj().load(Ordering::Relaxed))
+                .unwrap_or(0),
+        );
```

Comment:
Emmm... You need to modify `clone.rs`:
https://github.com/asterinas/asterinas/blob/bf6efbabc749167ea46b915d1ccac3871e8a0215/kernel/src/process/clone.rs#L401-L402